app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdScreen.kt

```diff
@@ -3,22 +3,62 @@ package org.simple.clinic.scanid
 import android.annotation.SuppressLint
```

Comment:
`Open search results when the QR code is scanned` 

to

`Open search results when a QR code is scanned`?